src/main/java/ch/njol/skript/effects/EffGlowingText.java

```diff
@@ -57,10 +67,19 @@ public class EffGlowingText extends Effect {
 
 	private boolean glowing;
 
+	@Nullable
+	private Side side;
```

Comment:
Is this tested on a version that has glowing text but _doesn't_ have the `org.bukkit.block.sign.Side` class?
I'd expect the class to fail to initialise if so, because it won't be able to resolve this field.

Reply:
Yeah forgot about that, thanks